### PR TITLE
Jonas pvalue readme

### PR DESCRIPTION
**Pull request by jmkuebler**
Computing p-values is the main usecase of our work. This is now described in the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,27 @@ or using the `install` target in the `Makefile` by simply running
     make install
 
 
+## Computing p-values
+If you want perform a two sample test on your own samples X and Y you can use the function `pvalue()` in 
+`tests-wo-split/methods/pvalue`.
+A simple test of the validity of our method is to see wheter the p-values are uniformly distributed 
+under the null hypothesis (samples come from the same distritbution).
+#####Example: uniform distribution of p-values
```
Comment (ioreshnikov) [inline README.md]: This doesn't work that well with github's `markdown` renderer :) If you remove this line at all, the rest would be parsed as a code block and would be rendered correctly.

File: README.md [hunk 2]
```diff
@@ -30,6 +30,27 @@ or using the `install` target in the `Makefile` by simply running
     make install
 
 
+## Computing p-values
```
Comment (ioreshnikov) [inline README.md:33]: I'd add a blank line here. We do that for every second-level header in the rest of the readme.

File: README.md (hunk 1 above)
Comment (jmkuebler) [reply README.md]: Sorry, I am not sure if I understand. Which line do you mean? 35? Could you implement it the way it should be? I was already satisfied with the way it was rendered :D

File: README.md (hunk 2 above)
Comment (jmkuebler) [reply README.md:33]: Ok, did it.

File: README.md (hunk 1 above)
Comment (jcpassy) [reply README.md]: Check here and you will see that it is not rendered properly:
https://github.com/MPI-IS/tests-wo-splitting/tree/jonas_pvalue_readmen

Comment (jcpassy) [reply README.md]: You need to remove a hash and add a space:
`#### Example:...`

Comment (jmkuebler) [reply README.md]: Ok, done.

**Comment by jcpassy** [comment]
All good, squashing and merging.

**Pull request merged**